src/content/docs/software/dubicube/group-level-sensitivity.md

```diff
@@ -105,6 +111,14 @@ library(b3gbi)        # Process occurrence cubes
 # General
 library(ggplot2)      # Data visualisation
 library(dplyr)        # Data wrangling
```

Comment:
You might want to use `library(dplyr, warn.conflicts = FALSE)`

Reply:
Thanks! I have updated this in dubicube v0.8.1 and looks good now, see [8b31af8](https://github.com/b-cubed-eu/documentation/pull/93/commits/8b31af8820d553f12ffab952eb6c2ed3930cd05c)